docs/tutorials/send_receive_ingress.md

```diff
@@ -103,76 +195,138 @@ Wait for the Broker to be ready:
 ```console
 kubectl get ActivemqArtemis  --watch
 NAME           READY   AGE
-send-receive   False   14s
-send-receive   True    51s
 send-receive   True    51s
 ```
 
-#### Create a route to access the ingress:
+Check that the ingress is available and has an IP address:
+
+```console
+kubectl get ingress --show-labels
+NAME                                 CLASS   HOSTS                                                                            ADDRESS         PORTS     AGE     LABELS
+send-receive-sslacceptor-0-svc-ing   nginx   send-receive-ss-0.send-receive-hdls-svc.send-receive-project.svc.cluster.local   192.168.39.68   80, 443   6m16s   ActiveMQArtemis=send-receive,application=send-receive-app,statefulset.kubernetes.io/pod-name=send-receive-ss-0
+```
+
+#### Make the cluster domain resolving to an IP address
+
+We will use networkmanager's dnsmasq plugin
+([source](https://fedoramagazine.org/using-the-networkmanagers-dnsmasq-plugin/))to
+configure the ip associated with the domain `cluster.local`. The dnsmasq plugin
+has wildcards which is better than setting manually every hosts in the
+`/etc/hosts` file. However if you're not running network manager you'll have to
+use this file as a fallback method.
+
+1. Create the following files:
+
+```console
+$ cat << EOF > /etc/NetworkManager/conf.d/00-use-dnsmasq.conf
+# This enabled the dnsmasq plugin.
+[main]
+dns=dnsmasq
+EOF
+```
+
+```console
+$ cat << EOF > /etc/NetworkManager/dnsmasq.d/00-cluster.local.conf
+# This file sets up the local local domain and
+# defines some aliases and a wildcard.
+local=/local/
+
+# The below defines a Wildcard DNS Entry.
+address=/.cluster.local/192.168.39.168
+EOF
+```
+
+```console
+$ cat << EOF > /etc/NetworkManager/dnsmasq.d/02-add-hosts.conf
+# /etc/NetworkManager/dnsmasq.d/02-add-hosts.conf
+# By default, the plugin does not read from /etc/hosts.
+# This forces the plugin to slurp in the file.
+#
+# If you didn't want to write to the /etc/hosts file.  This could
+# be pointed to another file.
+#
+addn-hosts=/etc/hosts
+EOF
+```
 
-Check for the ingress availability:
+2. restart NetworkManager:
 
 ```console
-$ kubectl get ingress --show-labels
-NAME                                 CLASS   HOSTS                                                                              ADDRESS         PORTS     AGE   LABELS
-send-receive-sslacceptor-0-svc-ing   nginx   send-receive-sslacceptor-0-svc-ing-send-receive-project.localresolvconf.com   192.168.39.57   80, 443   18h   ActiveMQArtemis=send-receive,application=send-receive-app,statefulset.kubernetes.io/pod-name=send-receive-ss-0
+$ sudo systemctl restart NetworkManager
```

Comment:
This solution to support wildcard domains is great but I'm not sure it should be part of this document because it is not a requirement and could not work in all environments. It could be another document and this document could point to it as better alternative to directly using /etc/hosts.

Reply:
Done